src/e2e/puppeteer/__tests__/cursor-move.ts

```diff
@@ -0,0 +1,44 @@
+import click from '../helpers/click'
+import clickThought from '../helpers/clickThought'
+import getEditingText from '../helpers/getEditingText'
+import paste from '../helpers/paste'
+import press from '../helpers/press'
+import waitForEditable from '../helpers/waitForEditable'
+
+vi.setConfig({ testTimeout: 20000, hookTimeout: 20000 })
+
+it('move cursor from formatted thought to first unformatted thought in descending order', async () => {
```

Comment:
Any reason you changed this to a puppeteer test? We really want to use RTL tests when possible since puppeteer tests are so slow.

Reply:
@raineorshine , I tried to implement using RTL but we can't run document.execCommand in RTL.

Reply:
toggleSort works in RTL but I need to check the bold thought goes to first of children in this test.